<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureMeasurementUi</class>
 <widget class="QDialog" name="ConfigureMeasurementUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>606</x>
     <y>10</y>
     <width>97</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="setDAQBtn">
      <property name="text">
       <string>Set DAQ Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setEntryBtn">
      <property name="text">
       <string>Set Entry Settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="AddDAQBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add DAQ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddODTBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add ODT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddEntryBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Entry</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="daqBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="title">
    <string>DAQ Settings</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="modeBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="timestampBox">
        <property name="text">
         <string>Timestamp</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="pidBox">
        <property name="text">
         <string>PID off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ctrBox">
        <property name="text">
         <string>CTR field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="alternatingBox">
        <property name="text">
         <string>Alternating</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Event channel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="eventBox">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Priority</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="priorityBox">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Prescaler</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="prescalerBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="directionBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Direction</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>42</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="DAQbutton">
        <property name="text">
         <string>DAQ direction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="stimButton">
        <property name="text">
         <string>STIM direction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="entryBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>291</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Entry Settings</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>281</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="addressInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Address Extension</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="addressExtInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="typeInput">
       <item>
        <property name="text">
         <string>byte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ubyte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>short</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ushort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>word</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uword</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>long</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ulong</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float64</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>SeriesColor</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="colorBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureMeasurementUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureMeasurementUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
